src/util/dom.js

```diff
@@ -0,0 +1,37 @@
+import { isString } from './string'
+
+/**
+ * @param {HTMLElement | Array | String} target
+ * @param {Document} document
+ */
+export const domArray = (target, document) => {
+  let targets = []
+  switch (true) {
+    case isString(target):
+      targets = Array.from(document.querySelectorAll(target))
+      break
+    case target instanceof Array:
```

Comment:
Can we use `Array.isArray(target)` here?
That makes the function more compatible for Array instances of other documents, e.g. an iframe.

Reply:
Today I learned: `Array.isArray(target)` exists!

Didn't know that before 😄 Gonna adapt my code.